frontend/src/container/ForgotPassword.jsx

```diff
@@ -51,62 +51,56 @@ export class ForgotPassword extends React.Component {
     const { props, state } = this
 
     return (
-      <section className='unLoggedPage forgotpassword primaryColorBg'>
-        <div className='container-fluid'>
-          <div className='row justify-content-center'>
-            <div className='col-12 col-sm-11 col-md-8 col-lg-6 col-xl-4'>
-              <Card customClass='forgotpassword__card'>
-                <CardHeader customClass='forgotpassword__card__header primaryColorBgLighten text-center'>
-                  {props.t('Forgot password')}
-                </CardHeader>
+      <section className='forgotpassword primaryColorBg'>
+        <Card customClass='forgotpassword__card'>
+          <CardHeader customClass='forgotpassword__card__header primaryColorBgLighten text-center'>
+            {props.t('Forgot password')}
+          </CardHeader>
 
-                <CardBody formClass='forgotpassword__card__body'>
-                  <div>
-                    <div className='forgotpassword__card__body__title'>
-                      {props.t('Did you forget your password ?')}
-                    </div>
+          <CardBody formClass='forgotpassword__card__body'>
+            <div>
+              <div className='forgotpassword__card__body__title'>
+                {props.t('Did you forget your password ?')}
+              </div>
 
-                    <InputGroupText
-                      parentClassName='forgotpassword__card__body__groupemail'
-                      customClass=''
-                      icon='fa-envelope-open-o'
-                      type='email'
-                      placeHolder={props.t('Email')}
-                      invalidMsg='Email invalide.'
-                      // isInvalid={this.state.inputLogin.isInvalid}
-                      value={state.backupEmail.value}
-                      onChange={this.handleChangeBackupEmail}
-                      onKeyDown={this.handleInputKeyDown}
-                      maxLength={512}
-                    />
+              <InputGroupText
+                parentClassName='forgotpassword__card__body__groupemail'
+                customClass=''
+                icon='fa-envelope-open-o'
+                type='email'
+                placeHolder={props.t('Email')}
+                invalidMsg='Email invalide.'
+                // isInvalid={this.state.inputLogin.isInvalid}
```

Comment:
is this useful to keep ?

Reply:
A good habit about comment is the following one: never comment code without explanation. The explanation is the following one:

- if the code must be kept commented in the code, then there is a good reason (and it must be explained)
- if it's a comfort, then it must be removed (the developer will be able to get the code back from old commits)
- if it's an error, then it must be fixed.

Reply:
it's invalid, I removed it